src/Tribe/Plugins_List.php

```diff
@@ -0,0 +1,126 @@
+<?php defined( 'WPINC' ) or die;
+
+if (! class_exists( 'Tribe__Plugins_List' ) ) {
+	/**
+	 * A list of Tribe's major plugins. Useful when encouraging users to download one of these.
+	 */
+	class Tribe__Plugins_List {
```

Comment:
@bordoni If you are wondering if a Plugin such as Events Calendar Pro could register this info when it activates: I wish, it would be a lot cleaner. But we need the list of plugins and their download URLs when those plugins are not active on the system. Did I understand correct?